api/private/cerbos/runtime/v1/runtime.proto

```diff
@@ -51,7 +51,8 @@ message RunnableResourcePolicySet {
       map<string, google.protobuf.Empty> roles = 4;
       Condition condition = 5;
       cerbos.effect.v1.Effect effect = 6;
-      Expr output = 7;
+      Expr output = 7 [deprecated = true];
+      Output output_info = 8;
```

Comment:
`output_conf` might be better. Technically, we can rename the old field (e.g. `output` -> `output_old`) and name this one `output`. It'd still be backward compatible because the field number hasn't changed and we don't use the JSON or text representations of this proto anywhere. However that'd probably cause some confusion so probably not worth it.

Reply:
Since there is no problem with the backward compatibility, I think we could rename the old field to the `output_old`. It will look a bit ugly with the `output_info` or `output_conf` name 😄 (well, no one sees it but you know...). WDYT?

Reply:
We'll have to test it to make sure that renaming the field doesn't break anything. I know it's technically possible but in practice there might be some corner cases 🤷🏽 

Maybe instead of `output_conf` we can call it `emit_output`. I think it works with the rest of the field names (`emitOutput.on.conditionNotMet`).